sdk/devcenter/azure-developer-devcenter/azure/developer/devcenter/_operations/_operations.py

```diff
@@ -4222,58 +3775,62 @@ def skip_action(  # pylint: disable=inconsistent-return-statements
 
         cls: ClsType[None] = kwargs.pop("cls", None)
 
-        request = build_dev_boxes_skip_action_request(
+        _request = build_dev_center_skip_dev_box_action_request(
             project_name=project_name,
+            user_id=user_id,
             dev_box_name=dev_box_name,
             action_name=action_name,
-            user_id=user_id,
             api_version=self._config.api_version,
             headers=_headers,
             params=_params,
         )
         path_format_arguments = {
             "endpoint": self._serialize.url("self._config.endpoint", self._config.endpoint, "str", skip_quote=True),
         }
-        request.url = self._client.format_url(request.url, **path_format_arguments)
+        _request.url = self._client.format_url(_request.url, **path_format_arguments)
 
-        _stream = False
+        _stream = kwargs.pop("stream", False)
         pipeline_response: PipelineResponse = self._client._pipeline.run(  # pylint: disable=protected-access
-            request, stream=_stream, **kwargs
+            _request, stream=_stream, **kwargs
         )
 
         response = pipeline_response.http_response
 
         if response.status_code not in [204]:
+            if _stream:
+                response.read()  # Load the body in memory and close the socket
             map_error(status_code=response.status_code, response=response, error_map=error_map)
             raise HttpResponseError(response=response)
 
         if cls:
-            return cls(pipeline_response, None, {})
+            return cls(pipeline_response, None, {})  # type: ignore
 
     @distributed_trace
-    def delay_action(
+    def delay_dev_box_action(
         self,
         project_name: str,
+        user_id: str,
         dev_box_name: str,
         action_name: str,
-        user_id: str = "me",
         *,
-        until: datetime.datetime,
+        delay_until: datetime.datetime,
```

Comment:
This was a modification for all languages?

Reply:
Yes, I believe .net asked for that change, but everyone agreed it should be updated.